<launch>
  <arg name="cam_name" value="camera" />

  <env name="GSCAM_CONFIG" value="v4l2src device=/dev/video0 ! video/x-raw-yuv,width=640,height=480 ! ffmpegcolorspace " />
  <node pkg="gscam" type="gscam" name="$(arg cam_name)">
    <param name="camera_name" value="$(arg cam_name)" />
    <param name="camera_info_url" value="file:///${ROS_HOME}/camera_info/asus_640_480.yaml" />
    <remap from="camera/image_raw" to="$(arg cam_name)/image_raw" />
  </node>
</launch>
